pandas/core/series.py

```diff
@@ -5395,10 +5396,8 @@ def _convert_dtypes(
         return result
 
     # error: Cannot determine type of 'isna'
-    # error: Return type "Series" of "isna" incompatible with return type "ndarray
-    # [Any, dtype[bool_]]" in supertype "IndexOpsMixin"
     @doc(NDFrame.isna, klass=_shared_doc_kwargs["klass"])  # type: ignore[has-type]
-    def isna(self) -> Series:  # type: ignore[override]
+    def isna(self) -> Series:
         return NDFrame.isna(self)
```

Comment:
`return super().isna()`?

Same question below.

Reply:
avoiding super bc i always have to check which parent class it refers to